pkg/infrastructure/services/prometheus.go

```diff
@@ -48,7 +48,7 @@ func configurePrometheus(client *client.Client, kogitoService v1alpha1.KogitoSer
 
 	prometheusAddOnAvailable, err := isPrometheusAddOnAvailable(kogitoService)
 	if err != nil {
-		return true, err
+		return true, nil
```

Comment:
why don't you return the err here ?

Reply:
Ah, good catch it should be returned. Some messup when merging SW PR